packages/bpk-component-navigation-bar/stories.js

```diff
@@ -107,4 +107,35 @@ storiesOf('bpk-component-navigation-bar', module)
         }
       />
     </div>
+  ))
+  .add('With icon', () => (
+    <div className={getClassNames('bpk-navigation-bar-story')}>
+      <BpkNavigationBar
+        id="test"
+        title={
+          <div style={{ display: 'flex', alignItems: 'center' }}>
+            <img
```

Comment:
Wouldn't it be easier to use `BpkIcon`. If users want to use something else they can but as an example I think we should use an icon.